### PR TITLE
OIDC add display_errors param

### PR DESCRIPTION
**Pull request by marcoam**

### CODE REVIEW

File: app/_hub/kong-inc/openid-connect/index.md [hunk 1]
```diff
@@ -348,6 +348,7 @@ Parameter ¹                             | Description
 `audience_claim`                        | The Name of the claim (or a path) where the audience can be found.
 `domains`                               | The domains to be verified against the `hd` claim.
 `max_age`                               | The `max_age` (in seconds) for the previous authentication, specifically the `auth_time` claim.
+`display_errors`                        | Display additional debugging information to the user when there's an error. Not recommended to be enabled in a production environmment.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/openid-connect/index.md:351]:
```suggestion
`display_errors`                        | Display additional debugging information to the user when there's an error. Not recommended to be enabled in a production environment.
```

File: app/_hub/kong-inc/openid-connect/index.md [hunk 2]
```diff
@@ -1259,6 +1261,14 @@ Default | Required
 * `86400`
 * `1800`
 
+#### config.display_errors
+
+By enabling this setting, consumers will be shown additional debugging information if they try and log on but run into an error. This setting should be treated with care and preferrably should not be enabled in Production environments.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/openid-connect/index.md:1266]:
```suggestion
You can enable this setting to show consumers additional debugging information if they try to log on but run into an error. 
> **Warning:** This setting should be treated with care. In most cases, it should not be enabled in Production environments.
```

**Comment by lena-larionova** [comment]
@marcoam can you take a look at the review suggestions? Thanks!

**Comment by marcoam** [comment]
> @marcoam can you take a look at the review suggestions? Thanks!

Done! Sorry for the delay and thanks for the reminder 👍

**Pull request merged**